proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,258 @@
+   may be held exclusively by these means.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is
+   responsible for all duties unless otherwise assigned.
+2. It elects the Board from a number of active members and can deselect board
+   members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges
+   the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of
+   the Association.
+6. Namely the annual accounts and the annual report have to be presented in
+   written format to the General Assembly for decision making and discharge
+   of the board. It calls on an auditor, who belong neither to the board nor
+   any other panel appointed by the board and who is not employed by the
+   association, in order to audit the accounting and annual report and
+   relate their results to the General Assembly. The auditors have access
+   to all of the Association’s accounting documents.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
+   elected for a 3-year term.
+
+   TODO: Define the role responsibilities.
+
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+3. The Board decides on all matters unless a General Assembly decision is
+   required. It implements the decisions taken by the General Assembly.
+4. Each Board member has the right to represent the Association.
+5. Should a board member quit while in office, the board appoints a
+   provisional board member on the basis of an internet vote. The provisional
+   board member remains in office until the next General Assembly takes place.
+   The General Assembly decides about the definite succession within the board.
+6. In case expulsion of a board member according to §4 (5) it is replaced by a
+   provisional board member.
+7. The Board can appoint a full-time director to manage current affairs.
+8. The director participates in General Assemblies and Board meetings.
+9. The Board can amend the Articles of Association for formal reasons as
+   required by authorities.
+
+## 9. By-Laws
+
+The by-laws govern all details of the Association’s activities not covered in
+the Articles of Association. The Board implements the by-laws following
+core member approval by internet vote.
+
+## 10. Meeting Minutes
+
+All decisions by the Board and General Assembly are documented in writing in
+German and available for members’ perusal and for the examination of the
+resolutions by the registry court.
+
+## 11. Internet vote
+
+The Association may conduct votes via the Internet. An Internet vote is valid
+after an announcement via email or messenger, and if a quorum of at least
+1/3 of the core members participates. The voting period must be at least two
```

Comment:
1/3 is too low imho.  I would prefer to put quorum at 50%+1.  being a core member is a significant responsibility and we should all be good about keeping up to date with mixxx business.

Reply:
1/3 is the bare minimum required by the German law for such vote. 

This case is significant:
Imagine we need a quick decision, but only 6 of 15 Members respond?  With the 1/3 rule the vote is valid. With a 1/2 rule we need to call for an extra general assembly. Where the same 6 people or even less can decide. 

What is your reason for proposing 1/2? 
I don't really mind, but let's pick the value for a reason.

Reply:
I like the idea of a "majority of a majority" making decisions.  With 15 members and the 1/3 rule, only 3 people could change Mixxx without the approval from the other 12 -- (5 to have a quorum, 3 is a majority)! That seems dangerous if a small group works in secret to pick a busy time when a lot of people aren't around.

With the higher bar, you'd need 5 out of 15 people (8 to reach a quorum, 5 to have a majority)

Reply:
OK, that makes sense.